src/test/java/features/239 Initial WinAppDriver Implementation.feature

```diff
@@ -45,4 +45,37 @@ Feature: 239 Implement WinAppDriver to automate windows
   	When I switch to the Encode DNA PopUp Window in the next window
   	Then I verify the Header contains the text "A New Popup Window"
   	When I switch to the Encode DNA New Tab Page in the previous window
-  	Then I verify the Header contains the text "Window Opened in a New Tab"
\ No newline at end of file
+  	Then I verify the Header contains the text "Window Opened in a New Tab"
+
+  @239E
+  Scenario: 239E Verify notepad window has changed after entering text using image comparison
+	Given I switch to the Notepad App
+	  And I click the Text Editor field
+	  And I clear the Text Editor field
+	  And I verify the Text Editor field is empty
+	  And I take a screenshot of the window
+    When I enter test in the Text Editor field
```

Comment:
tabbing

Reply:
Yeah I really gotta fix that. Lets meet up and fix up intellij's spacing problems in feature files.